### ISSUE TITLE
WalletConnect V2 support

### ISSUE THREAD

**Issue by silvestrst-crypto** [Uniswap/web3-react#708]
Hi everyone,

I would like to ask about uniswap WalletConnect v2 support plans/schedule. WalletConnect expects DApps to transition to WalletConnect v2 by March 2023 (if I am not mistaken), however, I couldn't find any issues/PRs regarding the uniswap transition.

Thanks

**Comment by silvestrst-crypto** [comment]
@NoahZinsmeister apologies for name dropping, decided to ping you as you are a major contributor. Is this an appropriate place to ask the above question?

**Comment by Daargajedan** [comment]
See: 
https://github.com/Uniswap/web3-react/pull/724

Someone is working on it!

**Comment by joshuajiangdev** [comment]
Regards https://twitter.com/pedrouid/status/1616106202420617217?s=46&t=JDgF-SWtxVVB7T1CeFNbrg , do we have an estimated timeline https://github.com/Uniswap/web3-react/pull/724 will be merged? The PR seems haven't been updated in a month.

**Comment by JoeyKhd** [comment]
+1 on this

Walletconnect states that all apps must update to v2 before February 28

https://medium.com/walletconnect/walletconnect-v1-0-sunset-notice-and-migration-schedule-8af9d3720d2e

**Comment by rixrix** [comment]
+1

**Comment by MarcoMandar** [comment]
yes, I was looking for this

**Comment by alekoisaev** [comment]
+1

**Comment by hpihkala** [comment]
+1

**Comment by grabbou** [comment]
Yes, just came here to let you know that we're very close to shipping support in a couple of days. Preview /alpha will be released soon, will be looking for your feedback before it goes stable.

**Comment by carmenjohnson992** [comment]
fe
